tools/wptserve/wptserve/ws_h2_handshake.py

```diff
@@ -0,0 +1,253 @@
+"""This file provides the opening handshake processor for the Bootstrapping
+WebSockets with HTTP/2 protocol (RFC 8441).
+
+Specification:
+https://tools.ietf.org/html/rfc8441
+"""
+
+from __future__ import absolute_import
+import os
+
+from mod_pywebsocket import common
+from mod_pywebsocket.stream import Stream
+from mod_pywebsocket.stream import StreamOptions
+from mod_pywebsocket import util
+from six.moves import map
+from six.moves import range
+
+# TODO: We are using "private" methods of pywebsocket. We might want to
```

Comment:
At least the exceptions are exposed in `mod_pywebsocket.handshake` via its `__init__.py`. We could do the same for other methods, too.

Reply:
Any thought about that @ricea ?

Reply:
I'd like to do the refactor of pywebsocket after we land this, so we know exactly what we need to export, and can work out the nicest way to export it.